commands/download.go

```diff
@@ -100,7 +100,7 @@ func waitUntilSupportBundleIsReady(ctx context.Context, client downloadSupportBu
 			}
 
 			log.Debug(fmt.Sprintf("Support bundle status: %s", sbStatus))
-			if sbStatus == "success" {
+			if sbStatus != "in progress" {
```

Comment:
´in progress´ or ´in_progress´?

Reply:
It is with a space (I looked it up to make sure).